### ISSUE TITLE
Support for Transmog

### ISSUE THREAD

**Issue by ChloeDawn** [ochotonida/artifacts#338]
It would be great to see support for [Transmog](https://modrinth.com/mod/transmog) - a mod for transmogrifying items (including hiding them). It would be serve as a better alternative to right-click cosmetic toggling on Fabric, for example. Currently artifacts do not function as expected with the mod - if transmogrified to be hidden, they still render when equipped in a trinket slot.

**Comment by ochotonida** [comment]
I agree that this is a better alternative, the right-click toggling always had been a bit of a temporary fix until Trinkets added a toggle or a better solution came along. However, it looks like Trinkets compatibility has already been implemented in Transmog for 1.21. If you're playing on an older Minecraft version, you should ask the Transmog dev to backport it.
